src/libraries/Microsoft.PowerFx.Core/Texl/BuiltinFunctionsCore.cs

```diff
@@ -187,7 +187,8 @@ internal class BuiltinFunctionsCore
         public static readonly TexlFunction RoundUpT = _library.Add(new RoundUpTableFunction());
         public static readonly TexlFunction Second = _library.Add(new SecondFunction());
         public static readonly TexlFunction Sequence = _library.Add(new SequenceFunction());
-        public static readonly TexlFunction Shuffle = _library.Add(new ShuffleFunction());
+        public static readonly TexlFunction Shuffle = _library.Add(new ShuffleFunction());
+        public static readonly TexlFunction ShowColumns = _library.Add(new ShowColumnsFunction());
```

Comment:
>public static readonly TexlFunction ShowColumns = _library.Add(new ShowColumnsFunction()); [](http://example.com/codeflow?start=8&length=90)

Does this mean we'll need a PA Client update when they update?

Reply:
Yes.